src/NUnitFramework/Directory.Packages.props

```diff
@@ -6,30 +6,30 @@
   <!-- Packages for used features -->
   <ItemGroup>
     <PackageVersion Include="System.Collections.Immutable" Version="6.0.0" />
```

Comment:
There was something about higher versions of this package which caused an issue for the tests last time around too. For now I've left alone as it is only used by our tests